apps/dashboard/src/components/workflow-editor/schema.ts

```diff
@@ -24,12 +24,10 @@ export const workflowSchema = z.object({
   description: z.string().max(MAX_DESCRIPTION_LENGTH).optional(),
 });
 
-export const buildStepSchema = (controlsSchema?: ZodValue) =>
-  z.object({
-    name: z.string().min(1).max(MAX_NAME_LENGTH),
-    stepId: z.string(),
-    ...(controlsSchema ? { controlValues: controlsSchema } : {}),
-  });
+export const stepSchema = z.object({
+  name: z.string().min(1).max(MAX_NAME_LENGTH),
+  stepId: z.string(),
+});
```

Comment:
Wasn't the controlSchema here used for code based workflows?

Reply:
Nope, it wasn't used at all as a struct. I just kept it for the two fixed input fields (name, stepId) of the Configure Step sidebar to have an identical behavior to Create Workflow sidebar.